### ISSUE TITLE
Adding steps in the execution report

### ISSUE THREAD

**Issue by pawankums** [Ahmed-Ali/Cucumberish#67]
@Ahmed-Ali Thanks for your framework. Is there any way by which we can list down the steps as well in the execution report. Right now i am using Xcpretty which is listing only feature and scenario names not the step names. Is there any feature?

**Comment by TitouanVanBelle** [comment]
Not sure it's possible at the moment but this could be implemented using Xcode 9's new activity feature:

```
XCTContext.runActivity(named name: String, block: (XCTActivity))
```

See https://developer.apple.com/videos/play/wwdc2017/409/ (26:52)

**Comment by brentleyjones** [comment]
Released as part of 1.3.0. 🎉
